### PR TITLE
Implement Django debug toolbar #155322125

### PR DESCRIPTION
**Pull request by wanjiku-carol**
**What does this PR do?**
Implements the Django debug toolbar

**Description of work done?**
- Pip install `django-debug-toolbar`
- Add configurations to `settings/dev.py` and `urls.py`

**How can this be manually tested?**
> $ python manage.py runserver

**Relevant pivotal tracker stories?**
[#155322125](https://www.pivotaltracker.com/story/show/155322125)

**Screenshots**
screen shot 2018-03-05 at 5 53 04 pm

### CODE REVIEW

**Comment by wanjiku-carol** [comment]
@egichuri @HawiCaesar please review

File: .gitignore [hunk 1]
```diff
@@ -103,3 +103,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
```
Comment (HawiCaesar) [inline .gitignore]: oops ? Were we to commit this file ?

Comment (wanjiku-carol) [reply .gitignore]: You mean that black space over there @HawiCaesar ?

Comment (HawiCaesar) [reply .gitignore]: Let me rephrase. Is the file `staticfiles` going to be in all our directories if this PR is merged and hence you adding it to .gitignore ?

Comment (wanjiku-carol) [reply .gitignore]: Yes @HawiCaesar . When you run `python manage.py runserver` it will autogenerate the static files when the PR is merged. They are over 600 files so that's why we can't have them here

Comment (egichuri) [reply .gitignore]: *`collectstatic`

File: settings/dev.py [hunk 2]
```diff
@@ -1,8 +1,21 @@
-from .base import *  # noqa: F403,F401
+from .base import *  # noqa: F403,F401, F405
```
Comment (egichuri) [inline settings/dev.py]: do we need to ignore 405 for this line?

File: .gitignore (hunk 1 above)
Comment (HawiCaesar) [reply .gitignore]: eh, alright keep that

**Comment by HawiCaesar** [comment]
LGTM
